src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,26 @@
+
+<!DOCTYPE html>
+<html>
+	<head>
```

Comment:
@jasminechau2 @vocatwk @lealgamb:
There's a mix of tabs and spaces for indentation in this file, which results in indentation looking incorrect.
Even though it's (mostly) personal preference, open source projects often pick a convention and stick to it. You can then change your editors settings to use one or the other and format the entire file accordingly. :)